client/src/components/AccessControl.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+
+export default (ComposedComponent) => {
```

Comment:
This higher-order function/component works as a "lock" on whatever component it is invoked on, checking the user's state for authorization at render-time and redirecting them if they are not authorized.
We will expand this component's functionality to also handle reauthorizing with the token on page reload.